src/metabase/api/setup.clj

```diff
@@ -200,6 +212,14 @@
     :link        "/admin/settings/slack"
     :completed   (configured :slack)
     :triggered   :always}
+   {:title       (tru "Setup embedding")
+    :group       (tru "Get connected")
+    :link        "/admin/settings/embedding-in-other-applications"
+    :completed   (or
```

Comment:
The specs of this logic are from figma: https://www.figma.com/file/UFay4YjWyUMyl23T1cXzeC/Meet-first-time-embedders-at-the-door-with-a-dashboard?type=design&node-id=1264-41762&mode=design&t=kepstwC5dDtNMEBt-4